test/e2e/specs/fakerp/loglevel.go

```diff
@@ -9,10 +9,17 @@ import (
 
 	"github.com/Azure/go-autorest/autorest/to"
 
+	"github.com/openshift/openshift-azure/pkg/cluster/updateblob"
 	"github.com/openshift/openshift-azure/test/clients/azure"
 	"github.com/openshift/openshift-azure/test/sanity"
 )
 
+// This test also fulfills the need to test upgrade reentrancy (Jira-221):
+// Create an OSA cluster
```

Comment:
In a follow-up I think we can take out the steps from these comments as they are already in the referenced jira ticket.